### PR TITLE
meson: relax ukify requirements

### PR DESCRIPTION
**Pull request by bluca**
Installing ukify.py doesn't require a working UEFI architecture, but only that the bootloader option is enabled (and python3). On Debian Arch: all packages (like python scripts) can theorethically be built on any builder with any architecture, so there's no guarantee that it will actually be an EFI-enabled architecture to do that package build.

Relax the requirement to check only for the config option.

### CODE REVIEW

**Review by yuwata — APPROVED** [review]
LGTM. But I'd like to wait for review by @keszybz and/or @medhefgo.

**Comment by medhefgo** [comment]
> On Debian Arch

What's Debian Arch? That sounds like an abomination. :joy_cat:

Also, are you tying to fix `apt build-dep -a foreign-arch systemd` by any chance?

I would say just drop that requirement altogether. I see value in having ukify on non-UEFI arches for producing images, no?

File: meson.build
Comment (medhefgo) [inline meson.build]: not disabled == allowed

**Comment by dnicolodi** [comment]
> What's Debian Arch?

That would be Debian `Arch: all` packages. Namely Debian packages that do not contain architecture dependent component, like Python modules (Python bytecompilation happens at install time on Debian) or scripts.

**Review by medhefgo — APPROVED** [review]

File: meson.build
Comment (yuwata) [inline meson.build]: Then, the message needs to be updated.

**Review by keszybz — APPROVED** [review]
LGTM.

**Comment by keszybz** [comment]
jammy-amd64: 13151s TEST-55-OOMD:                       FAIL     (180 s), unrelated.

**Pull request merged**
